modules/interactivelist.js

```diff
@@ -0,0 +1,17 @@
+const interactivelist = () => {
+  const allmarked = document.querySelectorAll('.check_box');
+  allmarked.forEach((list) => {
+    list.addEventListener('change', (e) => {
+      const { id } = e.target;
+      const todos = JSON.parse(localStorage.getItem('todolist')) || [];
+      const matched = todos.find((todo) => todo.index.toString() === id);
+      if (matched !== undefined) {
+        matched.completed = !matched.completed;
+        matched.index = id;
+      }
+      localStorage.setItem('todolist', JSON.stringify(todos));
```

Comment:
- [ ] As suggested by the previous reviewer, please make sure when a task is marked on the `todo list`, its status changes to completed in local storage 😄 

![bug](https://github.com/rabayarabu/To-Do-List/assets/99656687/e491c3e5-46ae-4a54-a57b-1e335f6a6cc5)

- [ OPTIONAL]  Please try adding tasks, mark them as completed, refresh the page and add more tasks and mark them as completed too, and check if your local storage keeps up.

- [ OPTIONAL] It is a good idea  to test your code before submitting for review 👍🏾